packages/gdl-frontend/server/index.js

```diff
@@ -15,15 +15,22 @@ const { GDL_ENVIRONMENT } = require('gdl-config');
 const glob = require('glob');
 const csp = require('helmet-csp');
 const { join } = require('path');
+const { readFileSync } = require('fs');
+const axios = require('axios');
 
 const routes = require('../routes');
 const {
-  publicRuntimeConfig: { REPORT_ERRORS, SENTRY_PUBLIC_KEY, SENTRY_PROJECT_ID },
+  publicRuntimeConfig: {
+    REPORT_ERRORS,
+    SENTRY_PUBLIC_KEY,
+    SENTRY_PROJECT_ID,
+    siteTranslationServiceUrl
+  },
   serverRuntimeConfig: { port }
 } = require('../config');
 const contentSecurityPolicy = require('./contentSecurityPolicy');
 
-const languages = glob.sync('locale/*/messages.js').map(f => f.split('/')[1]);
+const languages = glob.sync('locale/*/en.json').map(f => f.split('/')[1]);
```

Comment:
Prøver å formulere det vi diskuterte enklere. Om du setter både `id` og `defaultMessage` i  `<FormattedMessage />` så oppretter du 2 states som gjør det samme. For det vi vil er å fallbacke til engelsk når teksten ikke er satt og `id` tilbyr dette allerede ifølge react-intl sin fallback algoritme.